src/sparse_merkle_tree/in_memory_smt.rs

```diff
@@ -184,8 +185,8 @@ impl JsFunctionContext<'_> {
         Ok(())
     }
 
-    fn get_proof(&mut self) -> NeonResult<Proof> {
-        let raw_proof = self.context.argument::<JsObject>(2)?;
+    fn get_proof(&mut self, pos: i32) -> NeonResult<Proof> {
```

Comment:
I think `u8` could be better for `pos`, also we can use `into()` to convert it for `JSObject`